### PR TITLE
Dockerfile:Add LABEL MAINTAINER

### LINKED ISSUE #591: Replace MAINTAINER with LABEL MAINTAINER
The MAINTAINER instruction has been [deprecated](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated)
The `LABEL` instruction is a much more flexible version of this.

Replace `MAINTAINER` with a `LABEL MAINTAINER=`

**Would you like to work on the issue?**

Yes i would like to work on this issue

### PR DESCRIPTION
**Pull request by RaiVaibhav**
Fixes #591 

Changes: Replace `MAINTAINER` with `LABEL MAINTAINER`

### CODE REVIEW

**Comment by RaiVaibhav** [comment]
@Orbiter @mariobehling @chiragw15 @yukiisbored @DravitLochan @tstreamDOTh  @aditya98ak  @dynamitechetan  @madhavrathi Please Review it

**Comment by RaiVaibhav** [comment]
@yukiisbored  i don't know what causing problem here

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 FROM ubuntu:latest
-MAINTAINER Ansgar Schmidt <ansgar.schmidt@gmx.net>
+LABEL MAINTAINER="ansgar.schmidt@gmx.net"
```
Comment (chiragw15) [inline Dockerfile]: Change MAINTAINER to lowercase.

Comment (RaiVaibhav) [reply Dockerfile]: @chiragw15  Changed

**Pull request merged**